### DISCUSSION TITLE
How do I override the default highlight group?

### DISCUSSION THREAD

**Discussion by Daydreamer-riri** [folke/snacks.nvim#71] [Q&A]
I wanted to override the background of terminal, but when I set SnacksNormal in my config, the highlight group changed back to link FloatNromal when I typed <C-/>.

**Comment by folke** [comment]
If you want to make changes to h groups, you always should do that in a `ColorScheme` autocmd

**Comment by Daydreamer-riri** [comment]
Thank you for your reply. Here's what I wrote, but it doesn't work:
```lua
vim.api.nvim_create_autocmd({ "ColorScheme" }, {
  pattern = "*",
  callback = function()
    print("---------------------------------------------")
    vim.api.nvim_set_hl(0, "SnacksNormal", { guibg = nil })
  end,
})
```

What did I do wrong?

**Reply by folke** [reply]
` { guibg = nil }` is not valid. Check the docs.

**Reply by Daydreamer-riri** [reply]
`vim.api.nvim_set_hl(0, "SnacksNormal", { guibg = nil })` is works.  After I opened terminal at least once and entered ':lua vim.api.nvim_set_hl(0, "SnacksNormal", {guibg = nil})', my terminal became transparent.


https://github.com/user-attachments/assets/f348ce0e-d374-4ceb-b2ce-fcf28f9ef1e1

**Reply by Daydreamer-riri** [reply]
> ` { guibg = nil }` is not valid. Check the docs.

Sorry to ask again. Is there a way to make my bottom split terminal background transparent? I don't think this position is floating, and its highlight group link to FloatNormal is a bit strange.
